README.md

```diff
@@ -49,6 +49,8 @@ Further reading: [Progressive Enhancement with WebGL and React](https://medium.c
 
 `yarn add @14islands/r3f-scroll-rig @react-three/fiber three`
 
+> **Note** `@react-three/fiber` is optional here; you can use vanilla three.js if you prefer and import what you need from `@14islands/r3f-scroll-rig/powerups` and `@14islands/r3f-scroll-rig/scrollbar`.
+
```

Comment:
This should be expanded upon in the intro's "Framework agnostic" if correct.

Reply:
Actually, only the `@14islands/r3f-scroll-rig/scrollbar` is the exception here.  `r3f` and `three` is optional when using this target. React is always required as this is a React library - maybe that should be made more clear in the readme?

The main use case here is to be able to build a smoothscrolling website with for instance Next.js, without bundling three+r3f, and when the client says "oh by the way, we need this webgl thing" - we can just add `r3f` and `three`, import from the main target and keep going with minimal effort.

`@14islands/r3f-scroll-rig/powerups` always require `r3f` and `three` since it builds on top of these.